lib/elixir/lib/map_set.ex

```diff
@@ -171,6 +171,51 @@ defmodule MapSet do
     end
   end
 
+  @doc """
+  Returns a set with elements that are present in only one but not both sets.
+
+  ## Examples
+
+      iex> MapSet.disjoint(MapSet.new([1, 2, 3]), MapSet.new([2, 3, 4]))
+      MapSet.new([1, 4])
+  """
+  @spec disjoint(t(val1), t(val2)) :: t(val1) when val1: value, val2: value
+  def disjoint(%MapSet{map: left}, %MapSet{map: right}) do
+    {small, large} =
+      case map_size(left) > map_size(right) do
+        true -> {right, left}
+        false -> {left, right}
+      end
```

Comment:
Just out of curiousity: is this style preferred (in this case) over a `if` / `else` statement?

Reply:
Good catch I would go with a `if` here, less verbose, but they would be the same. :)